osgtest/tests/special_install.py

```diff
@@ -30,10 +30,15 @@ def test_02_install_packages(self):
         fail_msg = ''
         pkg_repo_dict = OrderedDict((x, core.options.extrarepos) for x in core.options.packages)
 
-        # HACK: Install slurm out of development if we're running 'All' tests.
+        # HACK: Install Slurm and osg-tested-internal out of development-like repos.
         # SOFTWARE-1733 may one day give us a generalized solution.
+        if float(core.osg_release()) > 3.4:
+            devops_repo = 'devops-itb'
+        else:
+            devops_repo = 'osg-development'
+
         if 'osg-tested-internal' in pkg_repo_dict or 'slurm' in pkg_repo_dict:
-            pkg_repo_dict.update(dict((x, ['osg-development']) for x in core.SLURM_PACKAGES))
+            pkg_repo_dict.update(dict((x, [devops_repo]) for x in core.SLURM_PACKAGES + ['osg-tested-internal']))
```

Comment:
does this mean you want to install `osg-tested-internal` out of `devops-itb`?

Reply:
Yup, it's noted as a devops package in the spreadsheet. And since the repo specified here is used in the yum command's `--enablerepo=` option, we should still get all the packages that `osg-tested-internal` requires